Gemfile

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# gem "rails"
```

Comment:
While it doesn't matter all that much in here, leaving commented code in the PR is considered an "oopsie"